.github/workflows/build-docker.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           vcpkg_sha_short=$(git rev-parse --short=8 HEAD)
           echo "vcpkg_sha_short=$vcpkg_sha_short" >> $GITHUB_OUTPUT
           docker_build_label=hpccsystems/platform-build-base-${{ inputs.os }}
-          echo "docker_tag=$docker_build_label:$vcpkg_sha_short" >> $GITHUB_OUTPUT
+          echo "docker_tag=$docker_build_label:${{ inputs.os == 'centos-7' && 'hpcc-platform-9.8.x' || '$vcpkg_sha_short' }}" >> $GITHUB_OUTPUT
```

Comment:
Not clear why this changed - it would be worth having a comment in the file to explain.

Reply:
Because we have to pin that label to the older 9.8.x build. If he kept the logic the same as everything else, we'd be reliant on a build with the current vcpkg_sha_short. And the current vcpkg version that we use for 9.8.x will fail for centos-7.  This looks like a pretty clean way to sidestep that problem and 'pin' the centos-7 builds to our older hpcc-platform-9.8.x tag that we have pushed.

Reply:
Further - this pattern `x == y && aaa || bbb` is equivalent to `x == y ? aaa : bbb`

Reply:
@GordonSmith I was looking for a comment in the yml file explaining *why* this needs to be special-cased, not what it was doing.